audformat/core/utils.py

```diff
@@ -212,6 +212,58 @@ def concat(
         return df
 
 
+def duration(
+        obj: typing.Union[pd.Index, pd.Series, pd.DataFrame],
+        *,
+        num_workers: int = 1,
+) -> pd.Timedelta:
+    r"""Duration of all entries present in the object.
+
+    The object might contain a segmented or a filewise index.
+    For a segmented index the duration is calculated
+    from its start and end values.
+    If an end value is ``NaT``
+    or the object contains a filewise index
+    the duration is calculated from the media file
+    by calling :func:`audiofile.duration`.
+
+    Args:
+        obj: object conform to
+            :ref:`table specifications <data-tables:Tables>`
+            and containing a segmented index
```

Comment:
I think we now also support filewise tables.

Reply:
I removed "and containing a segmented index"